build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Builds bolt-compatible packages from Python requirements, and copies
+# them back out to CWD.
+
+set -euo pipefail
+
+dir=`dirname $0`
+
+if [ -w /var/run/docker.sock ]; then
+    DOCKER=docker
+else
+    DOCKER='sudo docker'
+fi
+
+tmpdir='/tmp/bobdeps'
+
+mkdir -p $tmpdir
+cp $dir/Dockerfile $tmpdir
+
+if [ -f $PWD/oldrequirements.txt ]; then
```

Comment:
Why can't all these copies be in the Docker file?  (If you must, pass as variables into the file, or template them out.)